client/src/pages/Dash/DashPage.js

```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import * as THREE from "three";
 import { OrbitControls } from "three/examples/jsm/controls/OrbitControls";
 import { GLTFLoader } from "three/examples/jsm/loaders/GLTFLoader";
-import GLB from './brain.gltf';
-import textureSrc from './Br_color2_2k.jpg';
+import GLB from './brain.glb';
+import textureSrc from './Brain_texture1.jpg';
+import yellowTextureSrc from './Br_color_2K_yellow.jpg';
```

Comment:
@baljinder981  not a big deal now, but on the next commit I would change this variable name to FrontRegion